### DISCUSSION TITLE
drop your proxies

### DISCUSSION THREAD

**Discussion by Theblackmamba824** [3kh0/ext-remover#623] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
drop ur proxies in the comments and ill drop mine 2

**Comment by Theblackmamba824** [comment]
https://se.curly.ml/

**Comment by Theblackmamba824** [comment]
https://[rammerheadisa.realproxyservice.gq](https://rammerheadisa.realproxyservice.gq/)/

**Comment by Theblackmamba824** [comment]
https://[flipgrid.ga](https://flipgrid.ga/)/

**Comment by Theblackmamba824** [comment]
https://[typeracer.ml](https://typeracer.ml/)/

**Comment by Theblackmamba824** [comment]
https://task-yt-nebula-unblocker.glitch.me/

**Reply by obkduishinijf** [reply]
suspended

**Comment by Theblackmamba824** [comment]
https://holy-polite-muse.glitch.me/

**Reply by GGboogie** [reply]
Out of every single one of these proxies, this one works. Thanks bro.

**Reply by Theblackmamba824** [reply]
np

**Comment by Theblackmamba824** [comment]
https://birdfood.gq/

**Comment by CALR3F** [comment]
1000+ proxy links here: https://docs.google.com/document/d/1Zv0llC4f0SixWUgudqBE0vI8X4Whty9kIwmTNrZIunU/edit?usp=sharing

**Reply by GGboogie** [reply]
one proxy from the list that worked for me was https://12gft.xyz/
all the other links BEFORE this one didn't work for me though.

**Reply by GGboogie** [reply]
https://penrose-production-1b0b.up.railway.app/ says server error

**Reply by GGboogie** [reply]
https://64.227.120.231/ This Proxy works for me.

**Reply by GGboogie** [reply]
https://ballbang.madalin-stunt-cars-3.com/ This Proxy Works for me.

**Comment by piplup7575** [comment]
what about no

**Comment by thebossjeff** [comment]
[voh.fyi](url)
sorry bout that lmao

**Comment by pazza2234** [comment]
sorry, but I don't share my proxy online. because if the school finds the proxies you post, they get blocked. would not recommend getting proxies from online sites.

**Comment by KandyKaneMarvel** [comment]
If you go into the shadowgmes repository here on GitHub and then go into releases there are like 120+ unblocked links

**Reply by FancyBoiL1** [reply]
*GASP* nooooo doing our OWN work???? Nahhhhhhh
